src/Backend/Modules/Pages/Installer/Data/locale.xml

```diff
@@ -2,6 +2,11 @@
 <locale>
   <Backend>
     <Pages>
+      <item type="label" name="LinkCssClass">
+        <translation language="nl"><![CDATA[link CSS class]]></translation>
+        <translation language="en"><![CDATA[link CSS class]]></translation>
+        <translation language="pl"><![CDATA[link CSS class]]></translation>
```

Comment:
I don't think this is correct Polish

Reply:
Correct :-D
Something like "link do klasy CSS"? :)

Reply:
No clue, I don`t speak Polish :) if you`re not sure I would remove the translation

Reply:
Done :)